jdaviz/app.py

```diff
@@ -451,8 +452,11 @@ def get_data_from_viewer(self, viewer_reference, data_label=None,
 
                         if cls is not None:
                             handler, _ = data_translator.get_handler_for(cls)
-                            layer_data = handler.to_object(layer_data,
-                                                           statistic=statistic)
+                            try:
+                                layer_data = handler.to_object(layer_data,
+                                                            statistic=statistic)
+                            except IncompatibleAttribute:
```

Comment:
Looks awfully similar to glue-viz/glue-astronomy#39 but I think we're patching different things?

Reply:
Ah yes it does seem like the same thing. I think we do want to handle things better in glue-astronomy but I think we still want to raise an error there rather than silently fail, and we do want to catch those exceptions here.

Reply:
So I would suggest we go ahead with this PR, modify the glue-astronomy one to raise a better exception, and then update the exception we catch in jdaviz once glue-astronomy is released.